owl/robots/PR2.owl

```diff
@@ -302,6 +370,7 @@
     <!-- http://knowrob.org/kb/knowrob.owl#PR2 -->
 
     <owl:Class rdf:about="http://knowrob.org/kb/knowrob.owl#PR2">
+        <rdfs:subClassOf rdf:resource="http://knowrob.org/kb/PR2.owl#PerceptualAgent"/>
```

Comment:
shouldn't there be rather (or additionally) an axiom about the component, i.e., that pr2 has a vision component?